### ISSUE TITLE
Generating nonpeaks data

### ISSUE THREAD

**Issue by saninta0212** [kundajelab/chrombpnet#202]
How exactly does the nonpeaks file differ among the different folds? ie from the script it seems like train/test/valid info is not used for creating gc matched negatives. After generating the negatives file I am noticing all the chromosomes are still present in all the folds so do I have to create the nonpeaks file per fold or can I just use the same nonpeaks file for every fold?

**Comment by panushri25** [comment]
yes do create a non-peaks file for each fold, for testing we actually are just sampling 1:1 ratio of positives and negatives its only for training that you you using `neg_to_pos_ratio_train`
